src/index.css

```diff
@@ -1,6 +1,294 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat&family=Roboto+Slab&display=swap');
+
 body {
   margin: 0;
-  font-family: cursive;
+  padding: 0;
+  box-sizing: border-box;
+  background-color: #f5f6fa;
+}
+
+header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: var(--white-two);
+}
+
+.heading-nav {
+  display: flex;
+  align-items: center;
+  gap: 3rem;
+}
+
+nav {
+  display: flex;
+  gap: 2rem;
+}
+
+nav a {
+  text-decoration: none;
+  font-family: var(--Montserrat);
+  font-size: 0.813rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  color: #121212;
+  opacity: 0.5;
+}
+
+form {
+  margin-bottom: 4rem;
+  display: flex;
+  gap: 2rem;
+}
+
+input {
+  width: 40%;
+  padding: 1rem 0.5rem;
+  border-radius: 2px;
+  border: solid 1px var(--white);
+  background-color: var(--white-two);
+  font-size: 1rem;
+  font-family: var(--Montserrat);
+  font-weight: bolder;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.15px;
+  color: var(--very-light-pink);
+}
+
+button[type='submit'] {
+  width: 10%;
+  background-color: var(--azure);
+  border: none;
+  font-family: var(--RobotoSlab);
+  font-size: 0.813rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 0.5px;
+  text-align: center;
+  color: var(--white-two);
+}
+
+.active {
+  opacity: 1;
+}
+
+.Bookstores {
+  padding-left: 3rem;
+  font-family: var(--Montserrat);
+  font-size: 1.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: var(--azure);
+}
+
+.L-Panel {
+  margin: 2rem auto;
+  width: 85%;
+  padding: 2rem 4rem 1.625rem;
+  border-radius: 4px;
+  border: solid 1px var(--white);
+  background-color: var(--white-two);
+}
```

Comment:
### Your's Design
![image](https://user-images.githubusercontent.com/93039359/230605678-3fe6f6df-ac34-48fb-8769-2d2bf5177dc7.png)

### Zeplin's Design
![image](https://user-images.githubusercontent.com/93039359/230605764-58c2c4f3-0406-4d49-b35b-11f6cddd777b.png)

- [ ] It would be nice if you make your contents aligned from top to bottom to follow the zeplin's design as suggested by the previous reviewer 😊

Reply:
thanks for the review. please i need your help fixing this requirement.

Reply:
@Gmanlove Sure dear. You can reach out to me on slack by searching my name Hammas in the full-time-students channel or call me on zoom 👍

Reply:
I have sent you an invite on zoom